plugins/modules/cloudfront_distribution.py

```diff
@@ -1711,6 +1738,14 @@ def validate_origin(self, client, existing_config, origin, default_origin_path):
                 origin['custom_headers'] = ansible_list_to_cloudfront_list(origin.get('custom_headers'))
             else:
                 origin['custom_headers'] = ansible_list_to_cloudfront_list()
+            if 'origin_shield' in origin:
+                origin_shield = origin.get('origin_shield')
+                if origin_shield.get('enabled') is not None:
+                    if origin_shield['enabled']:
+                        origin_shield_region = origin_shield.get('origin_shield_region').lower()
+                        if origin_shield_region is None:
+                            self.module.fail_json(msg="origins[].origin_shield.origin_shield_region must be specified"
+                                                  " when origins[].origin_shield.enabled is true.")
```

Comment:
```suggestion
            if origin.get('origin_shield', {}).get('enabled', False):
                origin_shield_region = origin.get('origin_shield').get('origin_shield_region').lower()
                if origin_shield_region is None:
                    self.module.fail_json(msg="origins[].origin_shield.origin_shield_region must be specified"
                                            " when origins[].origin_shield.enabled is true.")
```

I'm not sure if this increases the readability ....

Reply:
It looks better indeed. You don't need to default on False on line 1741, the default (`None`) is fine.
line 1742, you should default on a string to avoid the TypeError if `.get('origin_shield_region')` returns `None`.
line 1743, no need to compare with `None`.